### ISSUE TITLE
`indent` rule are missing a problem at `else` when I'm using Stroustrup style.

### ISSUE THREAD

**Issue by mysticatea** [eslint/eslint#5218]
`indent` rule are missing a problem at `else` when I'm using Stroustrup style.
On `v2.0.0-rc.1`.

**Code:** test.js

```js
/* eslint indent: 2*/
{
    if (a) {
        foo();
    }
  else {
        bar();
    }
}
```

**Expected:**

```
$ eslint test.js --no-ignore --no-eslintrc

test.js
  6:2  error  Expected indentation of 4 space characters but found 2  indent

✖ 1 problems (1 errors, 0 warnings)
```

**Actual:**

No errors.
